_twig/docker-compose.yml/application.yml.twig

```diff
@@ -40,7 +40,5 @@
         @('services.app.environment'),
         @('services.app.environment_secrets'),
      ]), 2, 6) | raw }}
-{% if @('app.mount_volume') %}
     volumes:
       - .:/app
-{% endif %}
```

Comment:
if CI tests need data that was in /app as part of the build then this mount will make that inaccessible

Reply:
can you give an example? not sure I understand why that would be the case 🤔

Reply:
e.g. `go generate` generating go files in /app that are somehow used in tests

Reply:
https://github.com/inviqa/harness-go/blob/406ffa24ae18900320a03b717e9bae829a2cb876/docker/image/app/include/Dockerfile.base.twig#L47 isn't run on a volume, so I'm not sure I get what you mean :)

Reply:
I think it'd be safer to not mount the volume in pipeline

Reply:
ah yeah, you are right 🤦🏻 makes more sense not to mount though like you said 👍🏻